youtube_dl/extractor/yahoo.py

```diff
@@ -556,3 +560,161 @@ def _real_extract(self, url):
+        ], webpage, 'spaceid')
+        formats = []
+
+        if display_id == host:
+            # Headline page (w/ multiple BC playlists)
+            stream_urls_plists = re.findall(
+                r'YJNEWS\.LIVESTREAM\.SRC.+=(?:\s+)?["\'](?P<streamurl>[^"\']+plist=(?P<plist>\d+)[^"\']+)',
+                webpage)
+            if not stream_urls_plists:
+                # Manually build ('news.yahoo.co.jp', ...)
+                sbase_url = 'https://s.yimg.jp/images/media/video/player/html/embed_1.2.5.html?service=news&service_type=video&plist=%s&poster=%s&sp=%s&country=JP'
+
+                params = re.findall(
+                    r'["\']plist["\']:\s*["\'](?P<plist>[^"\']+)[^;]+?["\']poster["\']:\s*["\'](?P<poster>[^"\']+)[^;]+?["\']spaceId["\']:\s*["\'](?P<spaceid>[^"\']+)',
+                    webpage)
+                for p in params:
+                    stream_urls_plists.append((sbase_url % p, p[0]))
+
+            bc_url = 'http://players.brightcove.net/%s/%s_%s/index.html?playlistId=%s'
+            entries = []
+            for url, plist_id in stream_urls_plists:
+                embed_page = self._download_webpage(
+                    'https://s.yimg.jp/images/media/video/player/js/embed_1.0.9.min.js',
+                    plist_id,
+                    headers={
+                        'Authority': 's.yimg.jp',
+                        'Path': '/images/media/video/player/js/embed_1.0.9.min.js',
+                        'Referer': url,
+                    }, fatal=False)
+                account = self._search_regex(
+                    r'\w+\.video_attr\[(["\'])data-account\1\]\s*=\s*\1(?P<accountid>[\d]+)',
+                    embed_page, 'account id', group='accountid', fatal=False)
+                player = self._search_regex(
+                    r'\w+\.video_attr\[(["\'])data-player\1\]\s*=\s*\1(?P<playerid>[^"\']+)',
+                    embed_page, 'player id', group='playerid', fatal=False)
+                embed = self._search_regex(
+                    r'(["\'])data-embed\1\s*:\s*\1(?P<embed>[^"\']+)',
+                    embed_page, 'embed', group='embed', fatal=False)
+                if not all([embed_page, account, player,  embed]):
+                    continue
+                entries.append(bc_url % (account, player, embed, plist_id))
+
+            return self.playlist_from_matches(entries, space_id, title, ie='BrightcoveNew')
+
+        # Article page
+        app_id = 'dj0zaiZpPVZMTVFJR0FwZWpiMyZzPWNvbnN1bWVyc2VjcmV0Jng9YjU-'
+        content_id = self._search_regex(
+            r'<script[^>]+class=(["\'])yvpub-player\1[^>]+contentid=(?P<contentid>[^&"\']+)',
+            webpage, 'contentid', group='contentid', default=space_id)
+        # md5 hash of space_id + '_headlines.yahoo.co.jp'
+        ak = hashlib.md5('_'.join((space_id, host)).encode()).hexdigest()
+
+        json_data = self._download_json(
+            'https://feapi-yvpub.yahooapis.jp/v1/content/%s' % content_id,
```

Comment:
This does not look to be working for `space_id` fallback.

Reply:
I see what you mean. I removed the `space_id` fallback.